tests/client/test_api.py

```diff
@@ -228,6 +228,14 @@ def test_log_passing_empty_records_list(mocked_client):
         api.log(records=[], name="ds")
 
 
+def test_log_deprecated_chunk_size(mocked_client):
```

Comment:
maybe we can add an additional test w.r.t. to too large `batch_size`?

Reply:
Done